IndicatorFields/incidentfield-firstseenbyfeed.json

```diff
@@ -0,0 +1,45 @@
+{
+    "id": "indicator_firstseenbyfeed",
+    "version": -1,
+    "modified": "2020-02-06T16:34:43.859782506Z",
+    "sortValues": null,
+    "vcShouldIgnore": false,
+    "commitMessage": "",
+    "shouldCommit": false,
+    "name": "First Seen By Feed",
+    "prevName": "First Seen By Feed",
+    "ownerOnly": false,
+    "placeholder": "",
+    "description": "The first time the indicator was seen by the feed vendor.",
+    "cliName": "firstseenbyfeed",
```

Comment:
maybe firstseentimebyfeed

Reply:
The server builtin field names are firstseen, lastseen
lets stick to the same convention